src/doom/r_main.c

```diff
@@ -1007,6 +987,23 @@ R_PointInSubsector
 }
 
 
+// [crispy]
+static inline boolean CheckLocalView(const player_t *player)
+{
+  return (
+    // Don't use localview if the player is spying.
+    player == &players[consoleplayer] &&
+    // Don't use localview if the player is dead.
+    player->playerstate != PST_DEAD &&
+    // Don't use localview if the player just teleported.
+    !player->mo->reactiontime &&
+    // Don't use localview if a demo is playing.
+    !demoplayback &&
```

Comment:
Small thought: probably worth to check for `!demorecording` here as well. At the moment, while demo recording there is a small notable jiterring between frames while camera rotation, most likely brought by "short tics".

Reply:
Thanks for the review Julia. I noticed this issue too. I have a fix for the jittering on my local pc that I will clean up and push when I return home this weekend.